File: tests/fixtures/python/simple_module.py
"""A simple module with top-level functions and imports."""
import os
from typing import Optional

GLOBAL_CONSTANT = 42


def add(a: int, b: int) -> int:
    """Add two numbers."""
    return a + b


def multiply(a: int, b: int) -> int:
    return a * b


async def fetch_data(url: str) -> Optional[str]:
    return None

File: tests/fixtures/typescript/simple_module.ts
import { helper } from './helper';
import type { Config } from './types';

const GREETING = 'hello';

export function add(a: number, b: number): number {
  return a + b;
}

function multiply(a: number, b: number): number {
  return a * b;
}

export async function fetchData(url: string): Promise<string | null> {
  return null;
}
